src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -1497,8 +1497,9 @@ bool IAsyncResult.CompletedSynchronously
         /// </remarks>
         public static TaskFactory Factory { get; } = new TaskFactory();
 
+        internal static readonly Task<VoidTaskResult> s_cachedCompleted = new Task<VoidTaskResult>(false, default, (TaskCreationOptions)InternalTaskOptions.DoNotDispose, default);
         /// <summary>Gets a task that's already been completed successfully.</summary>
-        public static Task CompletedTask { get; } = new Task(false, (TaskCreationOptions)InternalTaskOptions.DoNotDispose, default);
+        public static Task CompletedTask { get; } = s_cachedCompleted;
```

Comment:
Shouldn't this now be:
```C#
public static Task CompletedTask => s_cachedCompleted;
```
rather than having an unnecessary additional field?

Reply:
Yes